components/AdminFunc/CreateClass.tsx

```diff
@@ -0,0 +1,189 @@
+            type="number"
+            id="freeSpots"
+            name="freeSpots"
+            value={inputs.freeSpots}
+            onChange={handleChange}
+            required
+          />
+        </label>
+        <label htmlFor="status">
+          Status
+          <select id="status" name="status" onChange={handleChange} required>
+            <option value="">-- Pick one --</option>
+            <option value="AVAILABLE">Available</option>
+            <option value="NOT AVAILABLE">Not available</option>
+          </select>
+        </label>
+        <label htmlFor="week">
+          Week of the Year
+          <input
+            type="number"
+            id="week"
+            name="week"
+            //the onchange works but display wrong
+            value={inputs.week}
+            onChange={handleChange}
+          />
+        </label>
+        <label htmlFor="day">
+          Day
+          <select id="day" name="day" onChange={handleChange} required>
+            <option value="">-- Pick one --</option>
+            <option value="Monday">Monday</option>
+            <option value="Tuesday">Tuesday</option>
+            <option value="Wednesday">Wednesday</option>
+            <option value="Thursday">Thursday</option>
+            <option value="Friday">Friday</option>
+            <option value="Saturday">Saturday</option>
+            <option value="Sunday">Sunday</option>
+          </select>
+        </label>
+        <label htmlFor="startTime">
+          Starts at:
+          <select
+            id="startTime"
+            name="startTime"
+            onChange={handleChange}
+            required
+          >
+            <option value="">-- Pick one --</option>
+            <option value="08:00">08:00</option>
+            <option value="09:00">09:00</option>
+            <option value="10:00">10:00</option>
+            <option value="11:00">11:00</option>
+            <option value="12:00">12:00</option>
+            <option value="13:00">13:00</option>
+            <option value="14:00">14:00</option>
+            <option value="15:00">15:00</option>
+            <option value="16:00">16:00</option>
+            <option value="17:00">17:00</option>
+            <option value="18:00">18:00</option>
+            <option value="19:00">19:00</option>
+            <option value="20:00">20:00</option>
+          </select>
+        </label>
+        {/* todo if there's time: add teacher schema and use map for options, add CRUD */}
+        <label htmlFor="teacher">
+          Teacher
+          <select id="teacher" name="teacher" onChange={handleChange} required>
+            <option value="">-- Pick one --</option>
+            <option value="Sam">Sam</option>
+            <option value="Michael">Michael</option>
+            <option value="Sophie">Sophie</option>
+            <option value="Mohamud">Mohamud</option>
+            <option value="Irene">Irene</option>
+            <option value="Guido">Guido</option>
```

Comment:
Where is teacher Zsoka? 😱😜

Reply:
Is it important for the handeling of the form that all names start with an uppercase?
Having an uppercase have, in this context, only for use human a contextual value. For handeling the form it does nothing and doesn't provide any additional value.

Also, I see that you're planning to store the teachers name in the database; that's good!
But think about duplicate names; what happens if there are two teachers with the same name? How can the user of the application distinguish?

Reply:
if there is a teacher database, it's easy to distinguish based on id on code side, and potentially full name on the client side. 

right, the casing doesnt matter as long as it matches what's expected in the backend. Can update.

Reply:
will update later if i dont move teachers into their own schema